src/5e-SRD-Equipment-Categories.json

```diff
@@ -656,6 +620,42 @@
 			{
 				"url": "/api/equipment/scholars-pack",
 				"name": "Scholar's Pack"
+			},
+			{
```

Comment:
Should the holy symbols be added here?

Reply:
The holy symbols are all already in the `adventuring-gear` equipment category, if that's what you're suggesting?

Reply:
Oh excellent. I think I was confused why we moved the other things around but it doesn't really matter.

Reply:
Oh excellent. I think I was confused why we moved the other things around but it doesn't really matter.

Reply:
Yeah, my fault really. I accidentally removed the foci from `adventuring-gear` instead of `standard-gear`. By the time I realised, it was too late to <kbd>CTRL</kbd>+<kbd>Z</kbd>, so just whacked them back in.